### PR TITLE
fix: second passthrough on responsiveness

### PR DESCRIPTION
**Pull request by renanmav**
**What changed? Why?**

Fix the layout of get started on small screen devices like Tablets and iPhones.

**Notes to reviewers**

**How has it been tested?**

https://github.com/coinbase/build-onchain-apps/assets/43140758/f9e315dc-b4c0-4bf7-ae1f-f8529418e522

### CODE REVIEW

File: apps/build-onchain-apps/src/components/header/Header.tsx [hunk 1]
```diff
@@ -37,13 +37,16 @@ function Header({ ghost }: HeaderProps) {
...
     >
-      <div
-        className={clsx(
-          'container fixed z-10 mx-auto inline-flex h-[72px] w-screen items-center justify-between',
-          'rounded-[50px] border border-stone-300 bg-white bg-opacity-10 p-4 backdrop-blur-2xl xl:w-[1280px]',
-        )}
-      >
-        <Menu />
+      <div className="container fixed z-10 h-[72px] px-8">
+        {/** maybe have a CSS variable for this px-8 ^ */}
```
Comment (renanmav) [inline apps/build-onchain-apps/src/components/header/Header.tsx:41]: I'm still learning tailwind

https://tailwindcss.com/docs/adding-custom-styles#resolving-ambiguities

Comment (Zizzamia) [reply apps/build-onchain-apps/src/components/header/Header.tsx:41]: > I'm still learning tailwind
same!

**Comment by renanmav** [comment]
Rebased and solved conflicts.

File: template/src/global.css [hunk 2]
```diff
@@ -43,7 +43,8 @@ html .font-inter {
 }
 
 body {
-  min-width: 320px;
+  /* A-HA FOUND YA! ψ( ` ∇ ´ )ψ */
+  /* min-width: 320px;  */
```
Comment (renanmav) [inline template/src/global.css:47]: damn you

File: template/src/hooks/useMediaQuery.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { useState, useEffect } from 'react';
+
+export function useMediaQuery(query: string) {
```
Comment (renanmav) [inline template/src/hooks/useMediaQuery.ts:3]: I need to write a test for this

File: template/src/hooks/useMediaQuery.ts [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    if (media.matches !== matches) {
+      setMatches(media.matches);
+    }
+    const listener = () => {
+      setMatches(media.matches);
+    };
+    // The signature '(callback: ((this: MediaQueryList, ev: MediaQueryListEvent) => any) | null): void' of 'media.addListener' is deprecated.ts(6387)
+    // lib.dom.d.ts(15085, 8): The declaration was marked as deprecated here.
+    media.addListener(listener);
+    return () => media.removeListener(listener);
```
Comment (renanmav) [inline template/src/hooks/useMediaQuery.ts:17]: Deprecated?

File: template/src/pages/home/Home.module.css [hunk 5]
```diff
@@ -22,6 +22,12 @@
   line-height: 85px;
   text-align: center;
   word-wrap: break-word;
+
+  /* Expected "context" media feature range notation (media-feature-range-notation)Stylelintmedia-feature-range-notation */
+  /* stylelint-disable-next-line media-feature-range-notation */
+  @media (max-width: 768px) {
+    font-size: 64px;
+  }
```
Comment (renanmav) [inline template/src/pages/home/Home.module.css:30]: What?

https://stylelint.io/user-guide/rules/media-feature-range-notation

Should I be using `>=`?

File: template/src/global.css (hunk 2 above)
Comment (Zizzamia) [reply template/src/global.css]: 😆

File: template/src/components/header/Menu.tsx [hunk 6]
```diff
@@ -1,18 +1,25 @@
...
 function Menu() {
   const [isMobileMenuOpen, setMobileMenuOpen] = useState(false);
+  const isSmallScreen = useMediaQuery('(max-width: 768px)');
 
   const handleMobileMenuClick = useCallback(() => {
     setMobileMenuOpen(!isMobileMenuOpen);
   }, [isMobileMenuOpen]);
 
+  const MenuTitle = useMemo(() => {
+    return isSmallScreen ? 'BOAT' : 'BUILD ONCHAIN APPS';
```
Comment (Zizzamia) [inline template/src/components/header/Menu.tsx:20]: ⛵️

File: template/src/pages/home/Home.module.css (hunk 5 above)
Comment (Zizzamia) [reply template/src/pages/home/Home.module.css:30]: whatever uncle stylelint says, we do. ahahaahah

File: template/src/pages/home/Home.module.css [hunk 7]
```diff
@@ -32,12 +38,23 @@
   line-height: 24px;
   text-align: center;
   word-wrap: break-word;
+
+  /* stylelint-disable-next-line media-feature-range-notation */
+  @media (max-width: 768px) {
```
Comment (Zizzamia) [inline template/src/pages/home/Home.module.css:43]: but yeah, I never seen media query written in the way you do. I alwasy so them with `>=`

File: template/src/hooks/useMediaQuery.ts [hunk 8]
```diff
@@ -0,0 +1,19 @@
+import { useState, useEffect } from 'react';
+
+export function useMediaQuery(query: string) {
```
Comment (Zizzamia) [inline template/src/hooks/useMediaQuery.ts:3]: Note for future PR, have docs.

**Review by Zizzamia — APPROVED** [review]
Brilliant

**Pull request merged**